src/main/scala/net/psforever/login/WorldSession.scala

```diff
@@ -504,6 +505,215 @@ object WorldSession {
     result
   }
 
+  /**
+    * Move an item from one container to another.
+    * If the source or if the destination is a kind of container called a `LockerContainer`,
+    * then a special procedure for the movement of the item must be respected.
+    * If the source and the destination are both `LockerContainer` objects, however,
+    * the normal operations for moving an item may be executed.
+    * @see `ActorRef`
+    * @see `Containable.MoveItem`
+    * @see `Container`
+    * @see `Equipment`
+    * @see `LockerContainer`
+    * @see `RemoveEquipmentFromLockerContainer`
+    * @see `StowEquipmentInLockerContainer`
+    * @see `TaskResolver`
+    * @param taskResolver na
+    * @param toChannel broadcast channel name for a manual packet callback
+    * @param source the container in which the item is to be removed
+    * @param destination the container into which the item is to be placed
+    * @param item the item
+    * @param dest where in the destination container the item is being placed
+    */
+  def ContainableMoveItem(
+                           taskResolver: ActorRef,
+                           toChannel: String,
+                           source: PlanetSideServerObject with Container,
+                           destination: PlanetSideServerObject with Container,
+                           item: Equipment,
+                           dest: Int
+                         ) : Unit = {
+    (source, destination) match {
+      case (locker: LockerContainer, _) if !destination.isInstanceOf[LockerContainer] =>
+        RemoveEquipmentFromLockerContainer(taskResolver, toChannel, locker, destination, item, dest)
+      case (_, locker: LockerContainer) =>
+        StowEquipmentInLockerContainer(taskResolver, toChannel, source, locker, item, dest)
+      case _ =>
+        source.Actor ! Containable.MoveItem(destination, item, dest)
+    }
+  }
+
+  /**
+    * Move an item into a player's locker inventory.
+    * Handle any swap item that might become involved in the transfer.
+    * Failure of this process is not supported and may lead to irregular behavior.
+    * @see `ActorRef`
+    * @see `AvatarAction.ObjectDelete`
+    * @see `AvatarServiceMessage`
+    * @see `Containable.MoveItem`
+    * @see `Container`
+    * @see `Equipment`
+    * @see `GridInventory.CheckCollisionsVar`
+    * @see `GUIDTask.RegisterEquipment`
+    * @see `GUIDTask.UnregisterEquipment`
+    * @see `IdentifiableEntity.Invalidate`
+    * @see `LockerContainer`
+    * @see `Service`
+    * @see `Task`
+    * @see `TaskResolver`
+    * @see `TaskResolver.GiveTask`
+    * @see `Zone.AvatarEvents`
```

Comment:
Are these generated by your IDE? Seems quite excessive to me.

Reply:
By hand.  I ask my autoformatter to do as little as possible since I inevitably fight with it and, being "auto-", it will generally win through subversion.  The most I expect of it is to indent properly.

I come from a school of thought that goes, "If you use it, you document it."  If you wanted to boil it down, you could probably trim it to "@see Zone.AvatarEvents" to indicate that this function uses the avatar event pipeline and "@see TaskResolver" to indicate that we use the task resolution system.  For now, I'm just compiling the list of entires that will be refined during a "do much later" meta-task that is going back and correcting all the comments for formal API generation.